doc/source/ray-core/internals/token-authentication.rst

```diff
@@ -0,0 +1,389 @@
+it only creates one when none is present.
+
+Kubernetes RBAC Integration
+---------------------------
+
+For Ray clusters running on Kubernetes, Ray can leverage Kubernetes RBAC to handle authentication
+and authorization. Kubernetes environments typically provide identity through Service Accounts or
+integrations such as webhook token authentication or OpenID Connect tokens.
+
+To enable Kubernetes-based authentication and authorization, set **RAY_ENABLE_K8S_TOKEN_AUTH**
+in addition to ``RAY_AUTH_MODE=token``. When enabled, Ray will not validate tokens against a
+single static token, but rather make two API calls to the Kubernetes API server:
+
+1. **TokenReview**: Authenticates a token to a known user.
+2. **SubjectAccessReview**: Checks whether a user or group can perform an action.
+
+TokenReview
+~~~~~~~~~~~
+
+Upon receiving a request containing a token, Ray validates tokens by calling the ``TokenReview``
+API. All ``TokenReview`` requests from Ray specify the ``ray.io`` audience to prevent token
+misuse. Example request:
+
+.. code-block:: yaml
+
+   apiVersion: authentication.k8s.io/v1
+   kind: TokenReview
+   spec:
+     audiences:
+       - ray.io
+     token: a0AXooCgfzVabfasadftbNJ_4hl5556344534fZU0GsDlj...
+
+If the token is from an authenticated user, Ray receives a response with the following status:
+
+.. code-block:: yaml
+
+   apiVersion: authentication.k8s.io/v1
+   kind: TokenReview
+   status:
+     audiences:
+       - ray.io
+     authenticated: true
+     user:
+       groups:
+         - system:authenticated
+         - myteam@example.com
+         - ray-admins
+         - ray-users
+       username: my-user@example.com
+
+SubjectAccessReview
+~~~~~~~~~~~~~~~~~~~
+
+After authentication, Ray uses the user/group information from the ``TokenReview`` status for
+authorization via ``SubjectAccessReview``:
+
+.. code-block:: yaml
+
+   apiVersion: authorization.k8s.io/v1
+   kind: SubjectAccessReview
+   spec:
+     user: my-user@example.com
+     resourceAttributes:
+       verb: ray:write-user
+       group: ray.io
+       resource: rayclusters
+       name: ray-cluster
+       namespace: my-team
+
+The ``SubjectAccessReview`` ensures that only authenticated users who are granted RBAC access to
+the RayCluster with custom verb ``ray:write`` are allowed. Access is granted using standard
+Kubernetes RBAC APIs (``ClusterRole``, ``Role``, ``RoleBinding``, etc.). The name and namespace
+of the Ray cluster used in the ``SubjectAccessReview`` request are stored in environment
+variables **RAY_CLUSTER_NAME** and **RAY_CLUSTER_NAMESPACE** configured by KubeRay.
+
+Raylet Identity with Kubernetes Service Accounts
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+By default, the identity of the Raylet is bound to the Service Account token of the Pod. However,
+the Raylet does not use the default token in ``/var/run/kubernetes.io/serviceaccount/token``.
```

Comment:
/var/run/kubernetes.io/serviceaccount/token

or

/var/run/kubernetes.io/service-account/token (with a dash?)

The example yaml has a dash in the mountPath.  Let's make sure they all agree.